neuroptica/optimizers.py

```diff
@@ -7,7 +7,8 @@
 import numpy as np
 
 from neuroptica.components import MZI, PhaseShifter
-from neuroptica.layers import OpticalMeshNetworkLayer
+from neuroptica.layers import OpticalMeshNetworkLayer, Activation
+from neuroptica.nonlinearities import TrainableNonLinearity
 from neuroptica.losses import Loss
 from neuroptica.models import Sequential
 from neuroptica.utils import pbar
```

Comment:
please add the nonlinearity training to the `InSituGradientDescent` optimizer as well